### DISCUSSION TITLE
Disable stenographer

### DISCUSSION THREAD

**Discussion by DashCarlos** [Security-Onion-Solutions/securityonion#2276] [Unsupported Versions] [answered]
In the documentation there is the following information on how to disable stenographer:

![imagen_2020-12-09_165017](https://user-images.githubusercontent.com/31094207/101652507-9d842a80-3a3e-11eb-872b-ea3144a9959f.png)

But in global.sls there is no line that mentions steno, 
I guess I should include it like this? Thanks for the help
```
steno:
    enabled: False
```

**Comment by weslambert** [comment, marked as answer]
Try editing this value from the minion pillar

Thanks,
Wes
